framework/test/fixtures/config/mainnet/config.json

```diff
@@ -133,6 +133,10 @@
 					"0": {
 						"start": 1,
 						"end": 6901027
+					},
+					"1": {
+						"start": 6901027,
+						"end": 15000000
```

Comment:
I assume this will be changed before release. Do we have an issue created for that already?
It would be nice to reference the issue in this PR